canvas_modules/harness/cypress/e2e/canvas/comments-wysiwyg.cy.js

```diff
@@ -0,0 +1,76 @@
+describe("Test for verifying WYSIWYG comments functionality in the application", function() {
+	beforeEach(() => {
+		cy.visit("/");
+		cy.setCanvasConfig({ "selectedWYSIWYGComments": true });
+		cy.rightClickToDisplayContextMenu(500, 50);
+		cy.verifyOptionInContextMenu("New WYSIWYG comment");
+		cy.clickOptionFromContextMenu("New WYSIWYG comment");
+		cy.editTextInComment("", "First Comment", false);
+	});
+
+	it("Test to verify functionality of font option in WYSIWYG toolbar", function() {
+		cy.clickTextToolbarOption("submenu-font", "IBM Plex Mono");
+		cy.clickCanvasAt(5, 5);
+		cy.verifyWysiwygCommentStyles({ fontFamily: "IBM Plex Mono" });
```

Comment:
Sorry I probably should have pointed this out before. Instead of hard coding the attribute name in the object can you code this something like this:
```
cy.verifyWysiwygCommentStyles({ styleName: "fontFamily": styleValue: "IBM Plex Mono" });
```
That way you can make the `verifyWysiwygCommentStyles` method more generalized.